esphome/components/display/display_buffer.h

```diff
@@ -308,7 +308,7 @@ class DisplayBuffer {
 
   virtual int get_width_internal() = 0;
 
-  void init_internal_(uint32_t buffer_length);
+  virtual void init_internal_(uint32_t buffer_length);
```

Comment:
```suggestion
  virtual void init_internal(uint32_t buffer_length);
```

protected methods are `_` _except_ when they're virtual or overriden